macro/run_trac_its.C

```diff
@@ -107,15 +118,14 @@ void run_trac_its(std::string path = "./", std::string outputfile = "o2trac_its.
   //===========================================
 
   //-------------------- settings -----------//
-  std::uint32_t roFrame = 0;
   for (int iEvent = 0; iEvent < itsClusters.GetEntries(); ++iEvent) {
```

Comment:
Strictly speaking, there should be no loop here as the tree contains 1 entry with vector of clusters for all events/ROFs.
If for some reason we would write multiple entries, this code would fail: the ROFrecord actually gives the coordinate 
of the ROF data including the entry to read (currently 0). I would do here instead of the loop
````
assert(itsClusters.GetEntries()==1);
itsClusters.GetEntry(0);
````
````

Reply:
OK, but this will crush if the continuous mode clusters are produced with ``run_clus_itsSA.C``, storing one ROF per entry.